dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -4993,6 +4994,9 @@ bundles.item.less.show= View Less
 bundles.item.all.show= Showing all the assets
 bundles.item.loading = Loading...
 
+contentType.errors.input.maxlength = The maximum number of characters supported by this field has been reached.
```

Comment:
Wouldn't it be better to add a `{0}` to replace it with the maximum number of characters allowed? It would look like:

```
contentType.errors.input.maxlength = The maximum number of characters supported by this field is {0}.
```